src/overlays/actors/ovl_Obj_Wind/z_obj_wind.c

```diff
@@ -5,6 +5,8 @@
  */
 
 #include "z_obj_wind.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
+#include "z64math.h"
```

Comment:
Is `#include "z64math.h"` needed?

Reply:
Assuming its not included globally somewhere it is. for `SQ()`